TodoApp/src/TodoApp.Contracts/Helpers/IServiceHelper.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace TodoApp.Contracts.Helpers
```

Comment:
This namespace is not very descriptive. You should never have a `>/dev/null` folder in your contracts. Imagine what other similar interfaces might be present and choose a better name, please.